source/manifest.json

```diff
@@ -1,4 +1,10 @@
 {
+  "applications": {
+    "gecko": {
+      "id": "theoldreader@knyar",
+      "strict_min_version": "53.0"
+    }
+  },
```

Comment:
We don't need to do any complex packaging; generating `.crx` is not needed, just copying stuff over to `.gitignore`d folder like `build` should be enough.

Though even better would be to also produce a version-stamped `zip` for upload.

The tricky question is making this build script cross-platform (yes, I do a lot on Windows, sadly). Perhaps using some build system like Grunt would make the _pain_ equal cross-platform.. 😛